application/src/main/java/org/thingsboard/server/service/ws/DefaultWebSocketService.java

```diff
@@ -843,6 +851,19 @@ private void sendPing() {
                 }));
     }
 
+    private void cleanupStaleSessions() {
+        wsSessionsMap.values().forEach(md -> {
+            String sessionId = md.getSessionRef().getSessionId();
+            if (!msgEndpoint.isOpen(sessionId)) {
+                log.info("Cleaning up stale session [sessionId: {}, tenantId: {}, userId: {}] ", sessionId,
```

Comment:
Use standard way of logging: "[tenantId][userId][sessionId] Cleaning up stale session"